src/client/sandbox/xhr.js

```diff
@@ -21,6 +23,10 @@ export default class XhrSandbox extends SandboxBase {
         this.corsSupported = typeof xhr.withCredentials !== 'undefined';
     }
 
+    static isOpenedXhr (obj) {
+        return obj[IS_OPENED_XHR];
+    }
+
```

Comment:
We can detect that xhr request is opened using `readyState` property.
After calling `xhr.open(...)` the `readyState` become equal 1.
So you can rewrite method isOpenedXhr as:
```javascript
static isOpenedXhr (obj) {
       return isXhr(obj) && obj.readyState >= 1 ;
 }
```